### PR TITLE
feat(nomad): EE-2508 proxy requests to nomad server

### PR DESCRIPTION
**Pull request by cmenginnz**

### CODE REVIEW

File: cmd/agent/main.go [hunk 1]
```diff
@@ -169,6 +170,15 @@ func main() {
 		if err != nil {
 			log.Fatalf("[ERROR] [main,nomad] [message: Unable to retrieve local IP associated to the agent] [error: %s]", err)
 		}
+
+		nomadConfig.NomadAddr = goos.Getenv(agent.NomadAddrEnvVarName)
```
Comment (deviantony) [inline cmd/agent/main.go:174]: Where are these variables coming from? Set by the user when deploying the agent or available in the environment as part of a Nomad installation?

File: cmd/agent/main.go [hunk 2]
```diff
@@ -169,6 +170,15 @@ func main() {
...
+		if nomadConfig.NomadAddr == "" {
+			log.Fatalf("[ERROR] [main,nomad] [message: Unable to retrieve environment variable NOMAD_ADDR]")
+		}
+
+		nomadConfig.NomadToken = goos.Getenv(agent.NomadTokenEnvVarName)
```
Comment (deviantony) [inline cmd/agent/main.go:179]: This one is not mandatory?

File: cmd/agent/main.go (hunk 1 above)
Comment (cmenginnz) [reply cmd/agent/main.go:174]: These two environment variables are set as part of a Nomad installation.

File: cmd/agent/main.go (hunk 2 above)
Comment (cmenginnz) [reply cmd/agent/main.go:179]: yes, it should be mandatory. Fixed.

**Review by ArrisLee — APPROVED** [review]
LGTM

**Pull request merged**